livestream/sessions.yml

```diff
@@ -0,0 +1,107 @@
+- current_title: "Opening"
+  current_presenter: "Our Networks"
+  next_title: "Wireless Toronto"
+  next_presenter: "Gabe Sawhney"
+  next_time: 2018-07-14 10:00
+
+- current_title: "Wireless Toronto"
+  current_presenter: "Gabe Sawhney"
+  next_title: "Our _stable_ networks"
+  next_presenter: "Grant Gallo and Jenny Ryan"
+  next_time: 2018-07-14 11:00
+
+- current_title: "Our _stable_ networks"
+  current_presenter: "Grant Gallo and Jenny Ryan"
+  next_title: "Building something together with local networks"
+  next_presenter: "Cecylia Bocovich and Lauren Lapidge"
+  next_time: 2018-07-14 11:30
+
+- current_title: "Building something together with local networks"
+  current_presenter: "Cecylia Bocovich and Lauren Lapidge"
+  next_title: "The hybrid step toward migrating to P2P: Adding P2P backend to current centralized apps"
+  next_presenter: "vmon"
+  next_time: 2018-07-14 12:00
+
+- current_title: "The hybrid step toward migrating to P2P: Adding P2P backend to current centralized apps"
+  current_presenter: "vmon"
+  next_title: "Aether: Distributing social networks without distributed consensus"
+  next_presenter: "Burak Nehbit"
+  next_time: 2018-07-14 13:00
+
+- current_title: "Aether: Distributing social networks without distributed consensus"
+  current_presenter: "Burak Nehbit"
+  next_title: "Fandom as an alternative network"
+  next_presenter: "Rea McNamara, Owen G. Parry and Maya Ben David"
+  next_time: 2018-07-14 14:30
+
+- current_title: "Fandom as an alternative network"
+  current_presenter: "Rea McNamara, Owen G. Parry and Maya Ben David"
+  next_title: "Keynote: Navigating Digital Futures"
+  next_presenter: "Nasma Ahmed"
+  next_time: 2018-07-14 18:00
+
+- current_title: "Keynote: Navigating Digital Futures"
+  current_presenter: "Nasma Ahmed"
+  next_title: "The artists network, the data-body, and the social-body"
+  next_presenter: "Jennifer Seaman Cook"
+  next_time: 2018-07-15 10:00
+
+- current_title: "The artists network, the data-body, and the social-body"
+  current_presenter: "Jennifer Seaman Cook"
+  next_title: "Holding Data Together: Decentralized patterns for stewardship"
+  next_presenter: "Brendan O'Brien"
+  next_time: 2018-07-15 12:00
+
+- current_title: "Holding Data Together: Decentralized patterns for stewardship"
```

Comment:
Can you make clear these are lightning talks in some way?

Reply:
I didn't want to further add to the long titles and didn't see much benefit to emphasize. If we add additional field I think it will add more complexity to the already tight https://github.com/ournetworks/2018/issues/68

Reply:
Perhaps if we take the wide lower third option, then I can pile more text to these titles?

Reply:
What about just prefixing title with :zap: or ⚡ ?

Reply:
Good idea ⚡